cmd/kubectl-package/validatecmd/validate.go

```diff
@@ -53,6 +54,8 @@ func NewCmd(validator Validator) *cobra.Command {
 			return fmt.Errorf("validating package: %w", err)
 		}
 
+		log.Println("Package validated successfully!")
```

Comment:
Please use OutOfStdout for producing output! https://pkg.go.dev/github.com/spf13/cobra#Command.OutOrStdout

Reply:
Okay

Reply:
Sorry, forgot to write down why :D

Reason is that `log.Println` simply prints to stdout. When a cobra command is invoked it is possible to specify `io.Writer` for stdout and stderr which makes testing much easier!

Reply:
TIL!